tasks/build_npm.ts

```diff
@@ -0,0 +1,33 @@
+import { build, emptyDir } from "https://deno.land/x/dnt@0.33.0/mod.ts";
+
+const VERSION = "0.1.0";
+
+await emptyDir("./npm");
+
+await build({
+  entryPoints: ["./mod.ts"],
+  testPattern: "**/*_test.ts",
+  outDir: "./npm",
+  // see JS docs for overview and more options
+  shims: {
+    deno: true,
+  },
+  package: {
+    // package.json properties
+    name: "pomo",
+    version: VERSION,
```

Comment:
Get the version string from the first argument.
```suggestion
    version: Deno.args[0],
```

Reply:
Also, be sure to edit `.github/workflows/publish.yaml` and pass the `$GITHUB_SHA` as the version string.

Reply:
Resolved by <https://github.com/EthanThatOneKid/pomo/pull/1/commits/e263226f5abbd1ce5d73c83cbceaff05a7b6886b>.